katas/helloworld/carlos/HelloWorld.js

```diff
@@ -0,0 +1,3 @@
+export function sayHello() {
```

Comment:
files under `helloworld/carlos` were not expected to be included in this PR. I will merge, but I'll remove for main branch